build.gradle

```diff
@@ -391,29 +398,27 @@ project(':core') {
   }
 
   checkstyle {
-    configFile = new File(rootDir, "checkstyle/checkstyle.xml")
     configProperties = [importControlFile: "$rootDir/checkstyle/import-control-core.xml"]
   }
-  test.dependsOn('checkstyleMain', 'checkstyleTest')
 }
 
 project(':examples') {
-  apply plugin: 'checkstyle'
   archivesBaseName = "kafka-examples"
 
   dependencies {
     compile project(':core')
   }
 
+  testJar {
+    enabled = false
+  }
+
   checkstyle {
```

Comment:
No, this is "re-configuring" the `configProperties` to use a special import control file since core is under a separate package structure and you can't have 2 package structures in a single import control file.